config/v1/tests/apiservers.config.openshift.io/KMSEncryptionProvider.yaml

```diff
@@ -0,0 +1,137 @@
+apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
+name: "APIServer"
+crdName: apiservers.config.openshift.io
+featureGates:
+- KMSEncryptionProvider
+tests:
+  onCreate:
+    - name: Should be able to create encrypt with KMS for AWS with valid values
+      initial: |
+        apiVersion: config.openshift.io/v1
+        kind: APIServer
+        spec:
+          encryption:
+            type: KMS
+            kms:
+              type: AWS
+              aws:
+                keyARN: arn:aws:kms:us-east-1:101010101010:key/9a512e29-0d9c-4cf5-8174-fc1a5b22cd6a
+                region: us-east-1
+      expected: |
+        apiVersion: config.openshift.io/v1
+        kind: APIServer
+        spec:
+          audit:
+            profile: Default
+          encryption:
+            type: KMS
+            kms:
+              type: AWS
+              aws:
+                keyARN: arn:aws:kms:us-east-1:101010101010:key/9a512e29-0d9c-4cf5-8174-fc1a5b22cd6a
+                region: us-east-1
+    - name: Should be able to create encrypt with KMS for AWS without region
```

Comment:
Should we really allow that? Looking at the plugin's code, if the region is not provided, the plugin will try to get the region from the ec2 metadata API, but it can be disabled and I don't know if we can guarantee that the plugin will always have the permissions to call it. Maybe it will be simpler for us to just require the region

Reply:
Got it, updated to make the region field required.